app/uk/gov/hmrc/eusubsidycompliancefrontend/services/BusinessEntityJourney.scala

```diff
@@ -47,23 +47,31 @@ case class BusinessEntityJourney(
     )
 
   def setAddBusiness(b: Boolean): BusinessEntityJourney =
-    this.copy(
-      addBusiness = addBusiness.copy(value = b.some))
+    this.copy(addBusiness = addBusiness.copy(value = b.some))
 
 }
 
 object BusinessEntityJourney {
 
+  val eoriPrefix = "GB"
+
   implicit val format: Format[BusinessEntityJourney] = Json.format[BusinessEntityJourney]
 
+  def isEoriPrefixGB(eoriEntered: String) = eoriEntered.take(2) === eoriPrefix
```

Comment:
NABD - `startsWith` does the same.

Reply:
OH! I meant the 'ignore this' comment on that one. But yeah. Ignore the other one too! haha